src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIProjectsService.java

```diff
@@ -103,9 +110,9 @@ public TableResponse getPagedProjects(TableRequest tableRequest, Boolean isAdmin
 	 * Get information about a project as well as permissions
 	 *
 	 * @param projectId - The project to get info for
-	 * @return {@link ProjectInfoResponse}
+	 * @return {@link ProjectDetailsResponse}
 	 */
-	public ProjectInfoResponse getProjectInfo(Long projectId) {
+	public ProjectDetailsResponse getProjectInfo(Long projectId) {
 		Project project = projectService.read(projectId);
```

Comment:
Related to the above comment we should have this throw an error if there was an issue getting the project details